cpmpy/transformations/reification.py

```diff
@@ -1,17 +1,27 @@
 """
-  Transformations regarding reification constraints.
+    Transformations regarding reification constraints.
 
-  There are three types of reification (BV=BoolVar, BE=BoolExpr):
-    - BV -> BE      single implication, from var to expression
-    - BV <- BE      single implication, from expression to var
-    - BE == BV      full reification / double implication (e.g. BV <-> BE)
+    There are three types of reification (BV=BoolVar, BE=BoolExpr):
 
-  Using logical operations, they can be decomposed and rewritten to each other.
+    +--------------+------------------------------------------------------------+
```

Comment:
same comment about tedious to maintain lines; can we stick to simple lists, or else the other way of making tables that is less tedious?